### PR TITLE
Sync deprecation flag from MDN, phase 2

### PR DESCRIPTION
**Pull request by saschanaz**
A checklist to help your pull request get merged faster:
- [x] Summarize your changes
- [ ] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)
- [ ] Data: if you tested something, describe how you tested with details like browser and version
- [ ] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)
- [x] Link to related issues or pull requests, if any: #6975

Again generated from https://github.com/saschanaz/browser-compat-data/compare/deprecated-fetch

I was lazy so I tweaked #6862 to get a diff list:

```
api.BaseAudioContext.createScriptProcessor
  deprecated status set to true
api.CanvasRenderingContext2D.addHitRegion
  deprecated status set to true
api.CanvasRenderingContext2D.drawWindow
  deprecated status set to true
api.CloseEvent.initCloseEvent
  deprecated status set to true
api.DataTransfer.mozClearDataAt
  deprecated status set to true
api.DataTransfer.mozGetDataAt
  deprecated status set to true
api.DataTransfer.mozItemCount
  deprecated status set to true
api.DataTransfer.mozSetDataAt
  deprecated status set to true
api.DataTransfer.mozTypesAt
  deprecated status set to true
api.DeviceProximityEvent
  deprecated status set to true
api.DeviceProximityEvent.max
  deprecated status set to true
api.DeviceProximityEvent.min
  deprecated status set to true
api.DeviceProximityEvent.value
  deprecated status set to true
api.FileReaderSync.readAsBinaryString
  deprecated status set to true
api.GlobalEventHandlers.onkeypress
  deprecated status set to true
api.HTMLFrameSetElement
  deprecated status set to true
api.MerchantValidationEvent
  deprecated status set to true
api.MerchantValidationEvent.MerchantValidationEvent
  deprecated status set to true
api.MerchantValidationEvent.complete
  deprecated status set to true
api.MerchantValidationEvent.methodName
  deprecated status set to true
api.MerchantValidationEvent.validationURL
  deprecated status set to true
api.Node.isSupported
  deprecated status set to true
api.Node.nodePrincipal
  deprecated status set to true
api.PaymentCurrencyAmount.currencySystem
  deprecated status set to true
api.PaymentRequest.merchantvalidation_event
  deprecated status set to true
api.PaymentRequest.onmerchantvalidation
  deprecated status set to true
api.Permissions.revoke
  deprecated status set to true
api.RTCIceCandidatePairStats.priority
  deprecated status set to true
api.RTCIceCandidatePairStats.readable
  deprecated status set to true
api.RTCIceCandidatePairStats.writable
  deprecated status set to true
api.SharedWorkerGlobalScope.applicationCache
  deprecated status set to true
api.TrackDefault
  deprecated status set to true
api.TrackDefault.TrackDefault
  deprecated status set to true
api.TrackDefault.byteStreamTrackID
  deprecated status set to true
api.TrackDefault.kinds
  deprecated status set to true
api.TrackDefault.label
  deprecated status set to true
api.TrackDefault.language
  deprecated status set to true
api.TrackDefault.type
  deprecated status set to true
api.TrackDefaultList
  deprecated status set to true
api.TrackDefaultList.TrackDefault
  deprecated status set to true
api.TrackDefaultList.TrackDefaultList
  deprecated status set to true
api.TrackDefaultList.length
  deprecated status set to true
api.URLUtilsReadOnly
  deprecated status set to true
api.UserProximityEvent
  deprecated status set to true
api.UserProximityEvent.near
  deprecated status set to true
api.Window.mozAnimationStartTime
  deprecated status set to true
```

### CODE REVIEW

File: api/CanvasRenderingContext2D.json [hunk 1]
```diff
@@ -115,7 +115,7 @@
           "status": {
             "experimental": true,
             "standard_track": true,
-            "deprecated": false
+            "deprecated": true
```
Comment (ddbeck) [inline api/CanvasRenderingContext2D.json:118]: I was spot checking these and found that `addHitRegion` has several children. Shouldn't they also be marked as deprecated? Though we'll need to make a note of which child features were marked deprecated, so we can update any pages (if needed).

**Review by ddbeck — COMMENTED** [review]
Thanks, @saschanaz! Spotted one thing. It's not necessarily a problem (we can merge without it), but thought it might be worth considering.

File: api/CanvasRenderingContext2D.json (hunk 1 above)
Comment (saschanaz) [reply api/CanvasRenderingContext2D.json:118]: Oh, it should probably be checked by linter. I'll filed an issue.

**Review by ddbeck — APPROVED** [review]
OK, that one took a while. Sorry about that, @saschanaz, but thank you very much!

**Pull request merged**